microsoft-edge/devtools-guide-chromium/accessibility/lighthouse.md

```diff
@@ -35,15 +35,25 @@ To audit a page using the Lighthouse tool:
 
 1. Go to the URL that you want to audit.
 
-1. In DevTools, select the **Lighthouse** tool.  Configuration options are displayed:
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
 
-   ![Lighthouse configuration options](./lighthouse-images/accessibility-lighthouse.png)
+1. In DevTools, on the Activity Bar, select the **Lighthouse** tab.  If that tab isn't visible, click the **More tools** (![More tools icon](./lighthouse-images/more-tools-icon.png)) button.
 
-1. For **Device**, select **Mobile** if you want to simulate a mobile device.  This option changes your user agent string and resizes the viewport.  This option can affect the audit results.
+   The **Lighthouse** tool configuration options are displayed:
 
-1. In the **Categories** section, select **Accessibility**.
+   ![The Lighthouse tool configuration screen](./lighthouse-images/accessibility-lighthouse.png)
 
-1. Click **Generate report**. After 10 to 30 seconds, DevTools displays a report.  The report gives tips on how to improve the accessibility of the page:
+1. Set the following configuration options:
+
+   * For **Mode**, select **Navigation**.
+   * For **Device**, select **Mobile** if you want to simulate a mobile device.  This option changes your user agent string and resizes the viewport.  This option can affect the audit results.
+   * In the **Categories** section, select **Accessibility**, and clear the other categories.
+
+   ![The configuration options to use](./lighthouse-images/configuration-options.png)
+
+1. Click **Generate report**.
```

Comment:
```suggestion
1. Click the **Perform an audit** (![The 'Perform an audit' icon](./lighthouse-images/perform-audit-icon.png)) button.
```
create png showing +.  Or, use plain text:
1. Click the **Perform an audit** (+) button.

Reply:
1. Click the **Analyze page load** button.